drivers/azure/azure.go

```diff
@@ -193,7 +193,6 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Name:   flAzureImage,
 			Usage:  "Azure virtual machine OS image",
 			EnvVar: "AZURE_IMAGE",
-			Value:  defaultAzureImage,
```

Comment:
I don't believe we should remove default here as we have sensible defaults for all other fields and providing no value to image field will result in provisioning failing where in previous versions it was using the default image (though likely failing if it was not overridden due to 18.04 image no longer available) - but it's a blanket change of behavior we don't want to happen nevertheless.

Reply:
I understand your point about removing the default image value, and you're right. it likely won't cause problems. However, I agree that having sensible defaults for other fields, like vmSize, is important because it's tied to the logic in the UI. For example, the [vmSizeAvailabilityWarning](https://github.com/rancher/ui/blob/master/lib/nodes/addon/components/driver-azure/component.js#L381) function depends on the selected location and VM size, and it shows a warning if the chosen size isn't available in a specific region.

I'm okay with keeping the default image for consistency with the other fields, to prevent any issues during provisioning, especially since the previous version relied on it...

Reply:
@skanakal , let's return the default please then.

Reply:
@snasovich I have reverted it to the default settings.